api-tests/data/assign-roles/assign-roles.js

```diff
@@ -0,0 +1,39 @@
+const utils = require('../../common/utils')
+
+const roleWithPermissions = {
+    "id": utils.getRandomString(7),
+    "permissions": [
+        {
+            "id": "create-person"
+        },
+        {
+            "id": "view-person"
+        }
+    ]
+}
+
+const createPerson = {
+    "prefix": "Ms",
+    "givenName": "Cookie",
+    "familyName": "Cookie"
+}
+
+const createSite = {
+    "name": utils.getRandomString(5),
+    "alias": utils.getRandomString(3),
+    "parentSiteId": "5d01f252-d41c-40f8-b3c1-4bd7282fb28e",
```

Comment:
Where are you getting this ID from? Should be dynamically generated at init time?

Reply:
that's right @alexbaxter I was about to come to you with regards to this. I am not sure if the 'init-services' will be run in the pr pipeline for me to pick it up from there. Or else I can just add another line of code to handle this.

Reply:
If you can reasonable assume the root site has been created, then do a GET on /sites and parse the ID out of that.